specs/cm/change-mgt-spec.html

```diff
@@ -932,149 +932,14 @@ <h3>Usage Identifiers</h3>
-                <td><em>deprecated</em> oslc_cm:relatedTestExecutionRecord</td>
-                <td>zero-or-many</td>
-                <td>False</td>
-                <td>Resource</td>
-                <td>Reference</td>
-                <td><code style="white-space: pre;">any</code></td>
-                <td>
-                  Related to a QM test execution resource. It is likely that the target resource will be an
-                  <code>oslc_qm:TestExecutionRecord</code>, but that is not necessarily the case.
-                </td>
-              </tr>
-              <tr>
-                <td><em>deprecated</em> oslc_cm:relatedTestCase</td>
-                <td>zero-or-many</td>
-                <td>False</td>
-                <td>Resource</td>
-                <td>Reference</td>
-                <td><code style="white-space: pre;">any</code></td>
-                <td>
-                  Related QM test case resource. It is likely that the target resource will be an
-                  <code>oslc_qm:TestCase</code>, but that is not necessarily the case.
-                </td>
-              </tr>
-              <tr>
-                <td><em>deprecated</em> oslc_cm:relatedTestPlan</td>
-                <td>zero-or-many</td>
-                <td>False</td>
-                <td>Resource</td>
-                <td>Reference</td>
-                <td><code style="white-space: pre;">any</code></td>
-                <td>
-                  Related QM test plan resource. It is likely that the target resource will be an
-                  <code>oslc_qm:TestPlan</code>, but that is not necessarily the case.
-                </td>
-              </tr>
-              <tr>
-                <td><em>deprecated</em> oslc_cm:relatedTestScript</td>
-                <td>zero-or-many</td>
-                <td>False</td>
-                <td>Resource</td>
-                <td>Reference</td>
-                <td><code style="white-space: pre;">any</code></td>
-                <td>
-                  Related QM test script resource. It is likely that the target resource will be an
-                  <code>oslc_qm:TestScript</code>, but that is not necessarily the case.
-                </td>
-              </tr>
-            </tbody>
-          </table>
-        </section>
-
-        <section id="changesFrom2">
-          <h2>Changes from 2.0</h2>
-          <p>
-            The following lists the significant vocabulary changes that were introduced in this specification. The
-            changes are all upward compatible additions and therefore do not introduce incompatibilities with version
-            2.0.
-          </p>
-          <ul>
-            <li>Added <em>oslc_cm:priority</em> and <em>oslc_cm:severity</em></li>
-            <li>Added new change management types</li>
-          </ul>
-        </section>
+        <p>
+          The following lists the significant vocabulary changes that were introduced in this specification. The changes
```

Comment:
Why is this new property added? unlike the other new properties, this does not seem to exist in the "deprecated" section.
It seems like a new change, as opposed to reverting old changes?

Reply:
Those were added because of common practice and not related to any deprecated properties. I see I may have removed too much in that changes from 2.0 section though.

Reply:
I put back the deprecated terms table, and only removed the QM properties.